### ISSUE TITLE
warning message with ansible servicebus collection " Azure API profile latest does not define an entry for ServiceBusManagementClient"

### ISSUE THREAD

**Issue by martin2176** [ansible/ansible_collections_azure#1429]
##### SUMMARY

Creating Azure servicebus namespace with ansible.
The namespace gets created without any issues, however it give a warning during ansible play
TASK [Create a namespace] *********************************************************************************************************************************************************
[WARNING]: Azure API profile latest does not define an entry for ServiceBusManagementClient

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

azure.azcollection.azure_rm_servicebus module 
##### ANSIBLE VERSION

```
ansible [core 2.15.8]
  config file = None
  configured module search path = ['/opt/harness-delegate/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/site-packages/ansible
  ansible collection location = /opt/harness-delegate/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.9.18 (main, Sep 22 2023, 17:58:34) [GCC 8.5.0 20210514 (Red Hat 8.5.0-20)] (/usr/bin/python3.9)
  jinja version = 3.1.3
  libyaml = True

```

##### COLLECTION VERSION

```
ansible-galaxy collection list azure.azcollection

# /usr/local/lib/python3.9/site-packages/ansible_collections
Collection         Version
------------------ -------
azure.azcollection 1.19.0


```

##### CONFIGURATION

```
bash-4.4# ansible-config dump --only-changed
CONFIG_FILE() = None


```

##### OS / ENVIRONMENT

Red Hat Enterprise Linux release 8.9 (Ootpa)

##### STEPS TO REPRODUCE

```
---
- hosts: localhost
  ignore_unreachable: true
  ignore_errors: True
  vars:
      resource_group: harnessngdemo
      location: 'East US 2'
      sbnamespace: namespace112
      topic: topic1
      subscription: subscription1
      returndatadirectory: Cic0iOoFQf2vFNp-IKa7GA
      sku: standard
      state: present
  tasks:
    - name: Ensure resource group exist
      tags: createresourcegroup
      block:
       - name: Check for resource group
         azure_rm_resourcegroup:
           name: "{{ resource_group }}"
           location: "{{ location }}"
         register: resourcegroupreturndata
      always:
       - name: Copy return data to file
         local_action:
          module: copy
          content: "{{ resourcegroupreturndata }}"
          dest: "/tmp/{{ returndatadirectory }}-resourcegroupreturndata"
    - name: Create a namespace
      tags: createnamespace
      block:
       - name: Create a namespace
         azure_rm_servicebus:
           name: "{{ sbnamespace }}"
           sku: "{{ sku }}"
           resource_group: "{{ resource_group }}"
           state: "{{ state }}"
         register: sbnamespacereturndata
      always:
       - name: Copy return data to file
         local_action:
          module: copy
          content: "{{ sbnamespacereturndata }}"
          dest: "/tmp/{{ returndatadirectory }}-sbnamespacereturndata"
    - name: Create a topic
      tags: createtopic
      block:
       - name: Create a topic
         azure_rm_servicebustopic:
           name: "{{ topic }}"
           namespace: "{{ sbnamespace }}"
           resource_group: "{{ resource_group }}"
         register: topicreturndata
      always:
       - name: Copy return data to file
         local_action:
          module: copy
          content: "{{ topicreturndata }}"
          dest: "/tmp/{{ returndatadirectory }}-topicreturndata"
    - name: Create a subscription
      tags: createsubscription
      block:
       - name: Create a subscription
         azure_rm_servicebustopicsubscription:
           name: "{{ subscription }}"
           topic: "{{ topic }}"
           namespace: "{{ sbnamespace }}"
           resource_group: "{{ resource_group }}"
         register: subscriptionreturndata
      always:
       - name: Copy return data to file
         local_action:
          module: copy
          content: "{{ subscriptionreturndata }}"
          dest: "/tmp/{{ returndatadirectory }}-subscriptionreturndata"


```

##### EXPECTED RESULTS

No warning message

##### ACTUAL RESULTS

Warning 
[WARNING]: Azure API profile latest does not define an entry for ServiceBusManagementClient

Servicebus still get created without issues

```
bash-4.4# ansible-playbook ansible-create-azure-servicebus.yml  --tags createnamespace
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [localhost] ******************************************************************************************************************************************************************

TASK [Gathering Facts] ************************************************************************************************************************************************************
ok: [localhost]

TASK [Create a namespace] *********************************************************************************************************************************************************
[WARNING]: Azure API profile latest does not define an entry for ServiceBusManagementClient
changed: [localhost]

TASK [Copy return data to file] ***************************************************************************************************************************************************
changed: [localhost]

PLAY RECAP ************************************************************************************************************************************************************************
localhost                  : ok=3    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

cat Cic0iOoFQf2vFNp-IKa7GA-sbnamespacereturndata
{"id": "/subscriptions/db73da55-ad6a-4d59-aeea-bba62c278f2a/resourceGroups/harnessngdemo/providers/Microsoft.ServiceBus/namespaces/namespace112", "name": "namespace112", "type": "Microsoft.ServiceBus/Namespaces", "location": "Central US", "sku": "standard", "provisioning_state": "Succeeded", "status": "Active", "created_at": "2024-02-01 20:18:17.057000+00:00", "updated_at": "2024-02-01 20:18:59.453000+00:00", "service_bus_endpoint": "https://namespace112.servicebus.windows.net:443/", "metric_id": "db73da55-ad6a-4d59-aeea-bba62c278f2a:namespace112", "changed": true, "warnings": ["Azure API profile latest does not define an entry for ServiceBusManagementClient"], "failed": false}

```

**Comment by Fred-sun** [comment]
@martin2176  Fixed in #1395, please update to the latest version and try again. Thank you!

**Comment by martin2176** [comment]
Thanks  upgrade of the ansibke azure collection fixed the problem
